### PR TITLE
Improving conversation with the perspective api

### LINKED ISSUE #6664: Improving Conversations with the Perspective API
#### Proposed title of article

Improving Conversations with the Perspective API.

#### Proposed article introduction

The Perspective API is a machine learning tool that is designed to rate a string of text (i.e. a comment) and provide you with a Toxicity Score, a number representing how toxic the text is. The framework learns by perceiving how huge number of online conversations have been moderated and attempts to improve the quality of debate and ensure individuals aren't put off from participating. To get started with using the API, you'll have to demand API access from their site. When you receive the email saying you have access, you'll have to sign in to the Google Developer Console and get your API key. Make your certifications with how much security you'd like and afterward you're prepared to get started. This article incorporates data concerning what highlights are right now accessible and what they're at last intended to accomplish.

#### Key takeaways 

* By the end of this article, the following is expected;

1. Getting a Score with cURL
2. Send An Ajax Request
3. The WordPress Way
4. Privacy and Censorship

#### Article quality

This article will be on point, simple to understand, well formatted and I'm sure users will be able to learn the procedure of Improving Conversations with the Perspective API. The code snippets in this article will be simple to understand and implement. I hope we will have a good ride all along.

#### References

N/A

### PR DESCRIPTION
**Pull request by Fidelmwangi**
#### Formatting and structure checks
- [X] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [X] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [X] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [X] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [X] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [X] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [X] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [X] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [X] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [X] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [X] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [X] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [ ] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [X] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [X] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [X] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

#### Topic suggestion this closes
This closes #6664

### CODE REVIEW

**Comment by hectorkambow** [comment]
@section-engineering-education/peer-reviewers  

Team is anyone available to do a review on this PR - it has been idle for about 20 days.

File: content/authors/fidel-mwangi/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Fidel Mwangi
+type: authors
```
Comment (Jethro-magaji) [inline content/authors/fidel-mwangi/index.md:3]: Please change your current avatar to a bit more professional image.

File: content/articles/improving-conversation-with-the-perspective-api/index.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    }
+  },
+  "languages": [
+    "en"
+  ]
+}
+```
+As a result of your request, you will receive a decimal number as the final result. The API will then return for that comment a score of 0.5, which indicates that the comment has a 50% chance of being toxic. It is possible to use this score to govern how a comment is handled and displayed to the end user by labeling it as spam or offering a channel that allows clients to show less or more negative comments, just as Google has done with their method of scoring.
+
+![Image-1](Image-1)
```
Comment (Jethro-magaji) [inline content/articles/improving-conversation-with-the-perspective-api/index.md:117]: Please change all image path to this format 
`![Allcar] (/engineering-education/introduction-to-unity-engine /allcar.png)`

File: content/articles/improving-conversation-with-the-perspective-api/index.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
+#### Introduction
+Toxicity Score is a numerical representation of how detrimental a piece of writing is, and the Perspective API is anticipated to provide you with that information by rating a line of text (such as a remark). When considering how many online discussions have been led, the technique is to reduce the type of debate so that individuals aren't discouraged from participating. Using this number, engineers and distributors can pass along feedback to experts and aid commentators in their evaluations. Viewpoint models have been providing scores for quite some time that indicate substantial harm, insult, disdain, character assault and danger.
+
+### Prerequisites
```
Comment (Jethro-magaji) [inline content/articles/improving-conversation-with-the-perspective-api/index.md:4]: This section should come after the "Table of content" section

**Review by Jethro-magaji — CHANGES_REQUESTED** [review]
Hi! @Fidelmwangi 
Well done for such an amazing article. However, there are some corrections you need to make.
Run your article on tools like Grammarly and ProWritingAid for better editing, this would save the time to review your work.
Please, add all the images needed and use a high-resolution free stock image for your hero image.

Here's a [link](https://github.com/section-engineering-education/engineering-education/pull/6210/commits/acc44893d4ec258f4e077b9ce8e3e2ca2b9d6a9f) to a PR with little or no revision to guide you on formatting. However, if you have any questions or need help, you can reach out to me on the EngEd Slack channel.
Thanks.

**Comment by lalith1403** [comment]
plagiarism check done ✔️

**Comment by hectorkambow** [comment]
@Fidelmwangi 

Any update on this PR ?

**Comment by Fidelmwangi** [comment]
@hectorkambow Hello sir, this PR will be ready by tomorrow im working on it

**Comment by Fidelmwangi** [comment]
@Jethro-magaji Hi, this article is good to go

**Comment by hectorkambow** [comment]
👀  

@Fidelmwangi 

Upon a glance this content looks too similar to [this article](https://css-tricks.com/improving-conversations-using-perspective-api/). 

Either way it seems this topic is well covered in other blog sites as shown above - we prefer to publish original and unique content.  👍

**Pull request closed without merging**